vision/google/cloud/vision/annotations.py

```diff
@@ -89,8 +89,11 @@ def from_api_repr(cls, response):
         annotations = {}
         for feature_type, annotation in response.items():
             curr_feature = annotations.setdefault(_KEY_MAP[feature_type], [])
-            curr_feature.extend(
-                _entity_from_response_type(feature_type, annotation))
+            entity = _entity_from_response_type(feature_type, annotation)
+            if isinstance(entity, list):
+                curr_feature.extend(entity)
+            else:
+                annotations[_KEY_MAP[feature_type]] = entity
```

Comment:
Updated in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2937/commits/bd7fd857439c66e2d01c966c6d0c45d0c6ad99dd.